### PR TITLE
Clarify support of multiple metrics in FAQ

### PR DESCRIPTION
**Pull request by levan-m**
### What does this PR do?

Clarify support of multiple metrics in the readme.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Describe your test plan

Write there any instructions and details you may have to test your PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -362,6 +362,11 @@ Finally, we have verification that the deployment was correctly autoscaled:
 
     Yes.
 
+- **Does WPA support multiple metrics?**
+
+    Yes, WPA can scale on multiple metrics and works similar to [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics). WPA evaluates each metric separately and proposes maximum number of replicas of all.
```
Comment (urseberry) [inline README.md:367]:
```suggestion
    Yes, WPA can scale on multiple metrics and works similar to [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics). WPA evaluates each metric separately and proposes the maximum number of replicas of all.
```
```suggestion
    Yes, WPA can scale on multiple metrics and works similar to [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics). WPA evaluates each metric separately and proposes the maximum number of replicas needed to support all metrics.
```

Comment (urseberry) [inline README.md:367]:
```suggestion
    Yes, WPA can scale on multiple metrics and works similar to [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics). WPA evaluates each metric separately and proposes the maximum number of replicas needed to support all metrics.
```

Comment (levan-m) [reply README.md:367]: Thanks for the feedback! The idea about selecting maximum is that if WPA evaluates metric1, metric2, metric3 and for each it calculates 10, 20, 30 replica proposals respectively, the final proposal will be 30. Could 'support all metrics' be interpreted as 60 in this case?

Comment (urseberry) [reply README.md:367]: Ah, I see. Perhaps this?

> WPA evaluates each metric separately and proposes the number of replicas associated with the metric that requires the largest number. For example, if WPA evaluates metric1, metric2, metric3 and for each it calculates 10, 20, 30 replica proposals respectively, the final proposal is 30.

**Review by urseberry — COMMENTED** [review]
If you don't like my updated wording, can you propose another one, or let me know what you don't like about it? "proposes maximum number of replicas of all" is not grammatically correct. It should be "proposes **the** maximum." And I believe that its meaning is not clear.

File: README.md (hunk 1 above)
Comment (urseberry) [inline README.md:367]:
```suggestion
    Yes, WPA can scale on multiple metrics and works similar to [HPA](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#scaling-on-multiple-metrics). WPA evaluates each metric separately and proposes the number of replicas associated with the metric that requires the largest number. For example, if WPA evaluates metric1, metric2, metric3, and for each it calculates 10, 20, 30 replica proposals respectively, the final proposal is 30.
```

**Review by urseberry — APPROVED** [review]
Thanks! Per our Slack discussion, I added my wording as a suggestion. I also went ahead and approved the PR so as not to hold up the process.

**Review by fanny-jiang — APPROVED** [review]

**Pull request merged**